### ISSUE TITLE
v0.2.7 'kubectl: command not found'

### ISSUE THREAD

**Issue by grempe** [rimusz/coreos-osx-gui-kubernetes-cluster#10]
Downloaded 0.2.7
removed app dir in my home dir
re-launched 0.2.7 and created cluster.

```
...
k8s nodes list:
/Applications/CoreOS k8s Cluster.app/Contents/Resources/first-init.command: line 186: kubectl: command not found
...
```

Full output here:
https://gist.github.com/grempe/c3312e913e50a3c0a6a4

I also see this same error if I then close all windows and launch OS Shell from menu bar app.

```
# in os shell launched by app
bash-3.2$ which kubectl
bash-3.2$
```

No kubectl installed in bin dir:

```
bash-3.2$ ls -la ~/coreos-k8s-cluster/bin/
total 16888
drwxr-xr-x 6 glenn staff     204 Mar 16 16:29 .
drwxr-xr-x 9 glenn staff     306 Mar 16 16:26 ..
-rwxr-xr-x 1 glenn staff 7378924 Oct 29 10:23 etcdctl
-rwxr-xr-x 1 glenn staff 9066128 Feb 27 14:26 fleetctl
-rwxr-xr-x 1 glenn staff  153888 Mar 16 16:26 gsed
-rwxr-xr-x 1 glenn staff  686304 Mar 16 16:26 wget
bash-3.2$
```

**Comment by grempe** [comment]
I then did an updates > Update kubernetes cluster and OS X kubectl from the app menu and saw the following result:

https://gist.github.com/grempe/c2208714e7a9e183f120

This seems to have now installed 'kubectl' locally.  But this script threw an error as well at the very end:

```
...
/Applications/CoreOS k8s Cluster.app/Contents/Resources/update_k8s.command: line 82: fleetctl: command not found
fleetctl list-units:
/Applications/CoreOS k8s Cluster.app/Contents/Resources/update_k8s.command: line 84: fleetctl: command not found
...
```

No good deed goes unpunished.  :-)

**Comment by rimusz** [comment]
@grempe thanks for find it out, now it for sure got fixed in v0.2.8

**Comment by grempe** [comment]
:+1:
